packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -137,14 +130,15 @@ export function useGridEditRows(apiRef: GridApiRef) {
     [options.isCellEditable],
   );
 
-  const setEditCellProps = React.useCallback(
+  const changeCellEditProps = React.useCallback(
```

Comment:
`setEditCellProps`, `changeCellEditProps` => pretty confusing

Reply:
Yeah, the names are a bit weird, we can discuss. Before #1824, some components were directly publishing `GRID_CELL_EDIT_PROPS_CHANGE`, but that is not good because TypeScript can't suggested the arguments. This new method is just a helper to publish the event. The actual logic of updating the state is in `setEditCellProps`. I can't just call `setEditCellProps` because the prop won't be called.